blog/2022-11-08-nushell-0.71.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Nushell 0.71
+author: The Nu Authors
+author_site: https://twitter.com/nu_shell
+author_image: https://www.nushell.sh/blog/images/nu_logo.png
+excerpt: Today, we're releasing version 0.71 of Nu. 
+---
+
+# Nushell 0.71
+
+Nushell, or Nu for short, is a new shell that takes a modern, structured approach to your commandline. It works seamlessly with the data from your filesystem, operating system, and a growing number of file formats to make it easy to build powerful commandline pipelines.
+
+Today, we're releasing version 0.71 of Nu. This release includes lots of overall polish.
+
+<!-- more -->
+
+# Where to get it
+
+Nu 0.71 is available as [pre-built binaries](https://github.com/nushell/nushell/releases/tag/0.71.0) or from [crates.io](https://crates.io/crates/nu). If you have Rust installed you can install it using `cargo install nu`.
+
+If you want all the built-in goodies, such as support for dataframes and SQLite databases, you can install it via `cargo install nu --features=extra`.
+
+As part of this release, we also publish a set of optional plugins you can install and use with Nu. To install, use `cargo install nu_plugin_<plugin name>`.
+
+# Themes of this release
+
+## New `++` operator as a short-hand to append to lists (merelymyself)
+https://github.com/nushell/nushell/pull/6766
+
+
+
+## General consistency accross commands and types has improved:
+### More iterating filter commands now support the `$in` variable consistently (webbedspace)
+https://github.com/nushell/nushell/pull/6951
+### `List`s finally support `upsert` (fdncred)
+https://github.com/nushell/nushell/pull/6932
+### `merge` can now also operate on records (webbedspace)
+https://github.com/nushell/nushell/pull/6919
```

Comment:
Add this:
"
Formerly, `merge`'s type signature only accepted, as an argument, blocks returning records instead of just plain records. This restriction, which conflicted with `append`'s signature, has now been lifted. It is also likely that, due to the fact that little expressive power is gained from having `merge` accept blocks returning records, that that part of its type signature will be removed in a future version.
"